cluster/kubemark/gce/config-default.sh

```diff
@@ -19,6 +19,8 @@
 # gce/util.sh script which assumes config filename), but if some things that
 # are enabled by default should not run in hollow clusters, they should be disabled here.
 
+# shellcheck disable=SC2034 # Variables sourced in other scripts.
```

Comment:
The majority of variables in this script come up as [SC2034](https://github.com/koalaman/shellcheck/wiki/SC2034) or _"foo appears unused. Verify it or export it." _

With that I think it's safe to just disable the check for the entire script.

Reply:
yeah this is a wonderful "config script" that configures another script by exporting envs. 🙃